compose/config/config_schema_v3.0.json

```diff
@@ -310,7 +310,8 @@
           "additionalProperties": false
         },
         "internal": {"type": "boolean"},
-        "labels": {"$ref": "#/definitions/list_or_dict"}
+        "labels": {"$ref": "#/definitions/list_or_dict"},
+        "priority": {"type": "number"}
```

Comment:
Unfortunately, we can't modify the 3.x schemas as they are frozen to conform to the shape they were released in in the Engine.

For the time being, it's fine to keep this as a 2.x feature.